plugins/woocommerce/includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -1589,19 +1589,19 @@ public function get_wizard_in_cart_payment_gateways() {
 	public function get_wizard_manual_payment_gateways() {
 		_deprecated_function( __CLASS__ . '::' . __FUNCTION__, '4.6.0', 'Onboarding is maintained in WooCommerce Admin.' );
 		$gateways = array(
-			'cheque' => array(
+			\WC_Gateway_Cheque::ID => array(
```

Comment:
Similar to a previous comment, throughout the changeset (there's another example on line 1604 where we reference `\WC_Gateway_COD`) we should be consistent in terms of where we do and don't qualify our class names.

```suggestion
			WC_Gateway_Cheque::ID => array(
```